__test__/add/delete-add.test.js

```diff
@@ -0,0 +1,56 @@
+// eslint-disable-next-line import/no-named-as-default
+import ToDoList from '../../src/modules/render.js';
+
+describe('ToDoList', () => {
```

Comment:
[OPTIONAL] I think your description text should have a more meaningful title than just ToDoList. This would help other developers understand the purpose of your tests

Reply:
Yes, sir ... We'll fix it in the next part. We really appreciate your suggestion 🙂🇻🇪🇪🇬🙏🤝